main.c

```diff
@@ -38,6 +38,16 @@ int main(int argc, char* argv[])
 	/*
 	 * TODO: Print if its a win or draw
 	*/
+	if(f==1)
+	printf(" Congrats X wins\n");
```

Comment:
Can you indent the contents of the if statement? Basically adding a tab in the if statement.

Also, since there are multiple if-else if statements here, I think it will be better to enclose them in
curly braces. I know that it will work just fine without it, still, it will be better to use braces when there
are >1 conditions. I will still leave it to @Progyan1997 to take the final call